service/biz/biz_utils.go

```diff
@@ -0,0 +1,136 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package biz contains the infra-agnostic business logic of Enrollz Service hosted by the switch owner infra.
+package biz
+
+import (
+	"context"
+	"crypto"
+	"crypto/rsa"
+
+	tpm12 "github.com/google/go-tpm/tpm"
+)
+
+// TPMKeyParms is the structure that contains the key parameters - TPM_KEY_PARMS from TPM 1.2 specification.
+type TPMKeyParms struct {
+	AlgID     tpm12.Algorithm // Algorithm identifier.
+	EncScheme uint16          // Encryption scheme identifier.
```

Comment:
All the uint values in TPM_* structures defined in TCP use big endian (network byte order), it might be helpful to have a comment -- even if at the top covering the entire file -- as to what byte order is used in all the uint fields in these go structs.